neo/Settings.cs

```diff
@@ -15,6 +15,7 @@ internal class Settings
         public IReadOnlyDictionary<TransactionType, Fixed8> SystemFee { get; private set; }
         public Fixed8 LowPriorityThreshold { get; private set; }
         public uint SecondsPerBlock { get; private set; }
+        public uint TransactionDifficult { get; private set; }
```

Comment:
I think that we don't need to establish a difficulty, we just compare user transactions... the ones starting with more zeroes get priority.